lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -350,14 +385,51 @@ class AwsCompileCloudFrontEvents {
 
             let behavior = {
               ViewerProtocolPolicy: 'allow-all',
-              ForwardedValues: {
-                QueryString: false,
-              },
             };
+            let shouldAssignCachePolicy = true;
             if (event.cloudFront.behavior) {
+              if (
+                event.cloudFront.behavior.ForwardedValues ||
+                event.cloudFront.behavior.MinTTL ||
```

Comment:
I think we need to ensure that `0` is also picked, so `event.cloudFront.behavior.MinTTL != null`